Automattic/pocket-casts-android
# Pride: Add new default icon

## Description

This adds pride icon as the default icon for the pride month. 

## Testing Instructions

1. Install `release/7.39` release build on a device
2. Change the default icon
3. Checkout this branch
4. Run the app on the device
5. ✅ The custom icon changed in step 2 should be displayed
6. Go to Settings > Appearance
7. ✅ The custom icon changed in step 2 should be selected
8. ✅ The Pride icon should be displayed after "Red Velvet" with the "Pride 2023" label
9. Tap on it
10. ✅ The icon should change
11. Fresh install the app from this branch
12. ✅ The pride icon should be displayed
6. Go to Settings > Appearance
7. ✅ The Pride icon should be the selected icon

## Instructions for 7.41

We'll revert back to the original icon in 7.41. In order to do so all that needs to be done is:

```
git revert 78f641f0d972af2b6215682639f498260570e00f
```

We'll keep the icon there in Settings though! 🏳️‍🌈

## Screenshots or Screencast 

<img width=300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/180fa718-0eab-4cc1-b3ff-831461d33b3b"/>


## Checklist
 
#### I have tested any UI changes...
